### ISSUE TITLE
Training & Test Set

### ISSUE THREAD

**Issue by AlexanderGeiger** [sintel-dev/Orion#43]
### Description

We should figure out which training/test split of the data makes most sense. 
We currently use the whole available data for training if the user does not specify a test_size, i.e. the model is trained with anomalies.
We might want to remove anomalies from the training set such that the model can learn the normal behavior. In the original paper the training set also does not contain anomalies.

**Comment by salman087** [comment]
How would we know in a real dataset where are the anomalies and be sure that the dataset does not contain any anomalies for a training a model.
